radio/src/opentx.cpp

```diff
@@ -2465,10 +2465,6 @@ uint16_t stackAvailable()
 
 void opentxInit(OPENTX_INIT_ARGS)
 {
-#if defined(DEBUG) && defined(USB_SERIAL)
-  // CoTickDelay(5000); // 10s
-#endif
```

Comment:
Maybe chang to USB_CDC mode selected?

Reply:
? It was dead code...

Reply:
hm yeah,  you are right the delay is commented out, yet I would leave that in since it allows you to have usb-serial connected and get the early bootmessages via usb.

Reply:
Ah, that was the goal... ok, can put it back

Reply:
I just looked to add it as a cmake option but it's broken since the runtime USB mode selection anyway, USB only starts once we're running and have cleared the warnings. I suggest leaving as is and taking care of restoring it separately if needed.